internal/driver/controllerserver_helper.go

```diff
@@ -272,6 +291,30 @@ func (cs *ControllerServer) createLinodeVolume(ctx context.Context, label string
 	return result, nil
 }
 
+// isEncryptionSupported is a helper function that checks if the specified region supports volume encryption.
+// It returns true or false based on the support for encryption in that region.
+func (cs *ControllerServer) isEncryptionSupported(ctx context.Context, region string) (bool, error) {
+	log := logger.GetLogger(ctx)
+	log.V(4).Info("Checking if encryption is supported for region", "region", region)
+
+	// Get the specifications of specified region from Linode API
+	regionDetails, err := cs.client.GetRegion(ctx, region)
+	if err != nil {
+		return false, errInternal("failed to fetch region %s: %v", region, err)
+	}
+
+	// Check if encryption is supported in the specified region
+	for _, capability := range regionDetails.Capabilities {
+		if capability == "Block Storage Encryption" {
```

Comment:
I'm a bit concerned about this string comparison here. We could run into some issues down the line. Might be worth it to use regex to look for encryption enablement?

Reply:
Look around online for better ways to do it, this seems like the most optimal method.